monitors/SyntheticPegMonitor.js

```diff
@@ -142,7 +157,7 @@ class SyntheticPegMonitor {
     });
 
     // If the volatility percentage is greater than (gt) the threshold send a message.
-    if (pricefeedVolatility.abs().gt(this.toBN(this.toWei(this.volatilityAlertThreshold.toString())))) {
+    if (pricefeedVolatility.abs().gt(this.toBN(this.toWei(this.pegVolatilityAlertThreshold.toString())))) {
```

Comment:
Changed to accommodate separation between `syntheticVolatilityAlertThreshold` & `pegVolatilityAlertThreshold`